tests/unittests/qml/tst_ChromeController.qml

```diff
@@ -246,47 +209,49 @@ ChromeController {
         }
 
         function test_load_event(data) {
-            // event types
-            var started = Oxide.LoadEvent.TypeStarted
-            var committed = Oxide.LoadEvent.TypeCommitted
-            var succeeded = Oxide.LoadEvent.TypeSucceeded
-            var stopped = Oxide.LoadEvent.TypeStopped
-            var failed = Oxide.LoadEvent.TypeFailed
-            var redirected = Oxide.LoadEvent.TypeRedirected
+            // WebEngineLoadRequest status enum
+            var started = WebEngineLoadRequest.LoadStartedStatus;
+            var succeeded = WebEngineLoadRequest.LoadSucceededStatus;
+            var failed = WebEngineLoadRequest.LoadFailedStatus;
 
-            controller.forceHide = data.forceHide
-            controller.forceShow = data.forceShow
-            controller.defaultMode = data.defaultMode
-            webviewMock.locationBarController.mode = data.initialMode
-            showSpy.clear()
+            controller.forceHide = data.forceHide;
+            controller.forceShow = data.forceShow;
+            controller.defaultMode = data.defaultMode;
+            webviewMock.locationBarController.mode = data.initialMode;
+            webviewMock.isFullScreen = data.isFullScreen;
 
             function test_sequence(sequence, modes) {
-                for (var i in sequence) {
-                    webviewMock.loadEvent({type: sequence[i]})
+                 for (var i in sequence) {
+                    
+                    console.log("data: " + JSON.stringify(data))
```

Comment:
console.log left in accidentally?

Reply:
I guess so, it was for debugging -> I'll remove it